docs/cli-tool.md

```diff
@@ -0,0 +1,79 @@
+# Yari's CLI Tool
+
+## Basic usage
+
+Install dependencies:
+
+```
+$ yarn
+```
+
+Run the cli tool:
+
+```
+$ yarn tool --help
+
+  cli.js 0.0.0
```

Comment:
Does `caporal` have a way to use a different name than the one from the command line? It'd be nice to use `tool` (or whatever else we decide) instead.